app/assets/javascripts/comfy/admin/cms/filters.js.coffee

```diff
@@ -0,0 +1,14 @@
+$(document).ready( ->
+  # Create a change event listener for each topic checkbox.
+  # This will add or remove the related substring in the content input tag.
+  ['WDPA', 'OECM', 'PAME'].forEach((el) =>
```

Comment:
You're using a hard-coded array here of categories, is it possible that we could pass an array to this function instead dynamically?

Reply:
But good call with the JavaScript to change the form.

Reply:
I've changed this now to be dynamic.